### ISSUE TITLE
If no documentation license is selected, the clause is still included with a blank space

### ISSUE THREAD

**Issue by step21** [contributoragreements/ca-cla-chooser#84]

**Comment by silverhook** [comment]
Good catch. Good question if we should handle this from the technical PoV (e.g. remove the clause in such cases) or legal PoV (add a generic clause for such cases).

**Comment by step21** [comment]
I'm opeen for both :)

**Comment by silverhook** [comment]
I’ll think about it.

**Comment by silverhook** [comment]
It may make sense to tackle this at the same time as https://github.com/contributoragreements/ca-cla-chooser/issues/93 to fix this documentation problem along with the problem of how to manage hardware, data, content (and _their_ documentation).

**Comment by silverhook** [comment]
This is still very much an open question (i.e. not attempted in the https://github.com/contributoragreements/ca-cla-chooser/pull/94/ pull request)
